tests/test_checks.py

```diff
@@ -115,3 +121,39 @@ async def test_trace_stall(
     assert resp.status == 200, await resp.text()
     end = time.monotonic_ns()
     assert (end - start) / 1e9 >= 0.8
+
+
+class CheckForTesting(Check):
+    name = "for_testing"
+    description = """check for testing"""
+    default_enabled = False
+
+    def check(self, headers: CIMultiDictProxy) -> None:
+        if "For-Testing" not in headers:
+            self.fail("For-Testing not found in headers")
+
+
+class TestChecks:
```

Comment:
nit: this doesn't need to be a class-based test, we're not using any features of the class

Reply:
OK. I wasn't sure if I could overwrite a fixture from a function based test or not